examples/seismic/tutorials/13_LSRTM_acoustic.ipynb

```diff
@@ -0,0 +1,11376 @@
+      "text/plain": [
+       "<Figure size 576x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
```

Comment:
The output of this cell is insanely giant, maybe reduce the verbosity of the operator? Or the steps?